src/components/AccountDetails.tsx

```diff
@@ -27,87 +41,87 @@ export default function AccountDetail({ openOptions }: Props): ReactElement {
+          : theme.palette.common.black
+      }
+    >
+      <DialogTitle onClose={onClose}>
+        <Typography variant="h3">{t("account")}</Typography>
+      </DialogTitle>
+      <DialogContent>
+        <Box
+          display="grid"
+          gridTemplateColumns="1fr 1fr"
+          gridTemplateRows="auto auto"
+          rowGap={2}
+        >
+          <Typography variant="body2" color="text.secondary">
             {t("connectedWith")}&nbsp;
             {connectorName}
-          </span>
-          <span className="label">{t("balance")}</span>
-          <div className="data">
+          </Typography>
+          <Typography variant="body2" color="text.secondary">
+            {t("balance")}
+          </Typography>
+
+          <Stack direction="row" spacing={1}>
             <Davatar
               size={24}
               address={account || ""}
               generatedAvatarType="jazzicon"
             />
-            <span className="address">
+            <Typography variant="subtitle1">
               {account && shortenAddress(account)}
-            </span>
+            </Typography>
             {account && (
-              <a
+              <Link
                 href={getEtherscanLink(account, "address")}
                 target="_blank"
                 rel="noreferrer"
               >
-                {/* link icon */}
-                <svg
-                  width="16"
-                  height="16"
-                  viewBox="0 0 16 16"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path d="M11.6667 11.6667H4.33333V4.33333H8V3H4.33333C3.59333 3 3 3.6 3 4.33333V11.6667C3 12.4 3.59333 13 4.33333 13H11.6667C12.4 13 13 12.4 13 11.6667V8H11.6667V11.6667ZM9.33333 2V3.33333H11.7267L6.17333 8.88667L7.11333 9.82667L12.6667 4.27333V6.66667H14V2H9.33333Z" />
-                </svg>
-              </a>
-            )}
-          </div>
-          <span className="data">{ethBalanceFormatted}&#926;</span>
-          <div className="buttonGroup">
-            {account && (
-              <Copy toCopy={account}>
-                <span className="textStyle">{t("copyAddress")}</span>
-              </Copy>
+                <LaunchIcon fontSize="inherit" />
+              </Link>
             )}
-          </div>
-          <div className="buttonGroup">
-            <button
-              className="textStyle"
+          </Stack>
+          <Box>
+            <Typography variant="subtitle1">
+              {ethBalanceFormatted}&#926;
+            </Typography>
+          </Box>
+          <Box display="flex" alignItems="center">
+            {account && <Copy toCopy={account}>{t("copyAddress")}</Copy>}
+          </Box>
+          <Box display="flex" alignItems="center">
+            <Button
               onClick={() => {
                 openOptions()
               }}
+              startIcon={<ChangeIcon />}
+              data-testid="changeAccount"
```

Comment:
This is a step in the right direction. If you name it something like `changeAccountBtn` it will avoid clashing if in the future there is another place in the app where there is a change account function.